<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventGuard</class>
 <widget class="QMainWindow" name="EventGuard">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EventGuard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/icons/icons/bhnam.ico</normaloff>:/icons/icons/bhnam.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_restart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_restart">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_systen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>560</width>
      <height>280</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="overview">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Overview</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="ruleSetting">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhNone</set>
     </property>
     <attribute name="title">
      <string>Rules</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_ruleBlock">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>255</width>
        <height>210</height>
       </rect>
      </property>
      <property name="title">
       <string>Block Rule</string>
      </property>
      <widget class="QLabel" name="label_ruleHost">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>IPv4</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ruleIP">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>190</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_rulePort">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rulePort">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_ruleName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>50</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ruleName">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>190</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ruleAdd">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>175</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Rule</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_ruleInbound">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Inbound</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_ruleOutbound">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Outbound</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_openWDF">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>175</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string> Open WDF</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_TCP">
          <property name="text">
           <string>TCP</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_UDP">
          <property name="text">
           <string>UDP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="checkBox_ruleAnyPort">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>70</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Any Port</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>270</width>
        <height>240</height>
       </rect>
      </property>
      <property name="title">
       <string>View Current Block Rules </string>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>20</y>
         <width>260</width>
         <height>180</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>205</y>
         <width>85</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="email">
     <attribute name="title">
      <string>Email</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="service">
     <attribute name="title">
      <string>Service</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="about">
     <attribute name="title">
      <string>About</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_bhnamssd">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>0</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>BHNamSSD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_check">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SyncSynchronizing"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_check">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exitApp">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>560</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>Log System</string>
    </property>
    <widget class="QListView" name="listView_logSystem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>540</width>
       <height>190</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
